### PR TITLE
fix: 'logs' route should always have :_id param

### LINKED ISSUE #47214: different api routes at `exercize tracker` tests challenge
### Describe the Issue

sometimes route url is `/api/users/:id/logs` and some other times it is `/api/users/:_id/logs` (`id` is `_id`)
this can be confusing (although the request is same)
### Affected Page

https://www.freecodecamp.org/learn/back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker

### Your code

```
app( `/api/users/:id/logs` , (req, res) => {
  res.send({success: true})
})
```

### Expected behavior

routes to be similar: `/api/users/:id/logs` 

### Screenshots

image

### System

- Device: [Laptop]
- OS: [Windows 10]
- Browser: [Chrome, FireFox]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by shootermv**
Checklist:

- [x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47214

### CODE REVIEW

File: curriculum/challenges/chinese/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md [hunk 1]
```diff
@@ -541,7 +541,7 @@ async(getUserInput) => {
 };
 ```
 
-你可以将 `from`、`to` 和 `limit` 参数添加到 `GET /api/users/:_id/logs` 请求检索任何用户的部分日志。 `from` 和 `to` 是 `yyyy-mm-dd` 形式的日期， `limit` 是希望返回的 log 数量。
+你可以将 `from`、`to` 和 `limit` 参数添加到 `GET /api/users/:id/logs` 请求检索任何用户的部分日志。 `from` 和 `to` 是 `yyyy-mm-dd` 形式的日期， `limit` 是希望返回的 log 数量。
```
Comment (Sboonny) [inline curriculum/challenges/chinese/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md:544]: Changes to different languages happen in crowdin, you should change the English file only

**Review by gikf — COMMENTED** [review]
I'd be inclined to use `_id`, as that's also the convention in tests body and on example project: https://exercise-tracker.freecodecamp.rocks.

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by moT01 — APPROVED** [review]
LGTM @shootermv 🎉 Thanks for contributing to freeCodeCamp 🎉

**Pull request merged**